### PR TITLE
DOC: merge_transformed_page()

### PR DESCRIPTION
**Pull request by paternal**
There is no example for `merge_transformed_page()`, although several issues could have been avoided if such an example existed (#1630, #1426, and issues that users do not bother to open). This pull request adds such an example.

Thanks for your work!

-- Louis

### CODE REVIEW

**Comment by pubpub-zz** [comment]
@paternal 
Thanks for you contribution. I would more likely recommend to use `.merge_transformed_page()` instead of doing the transform before merging. The advantage is that you can reuse the page.

**Comment by paternal** [comment]
> @paternal Thanks for you contribution. I would more likely recommend to use `.merge_transformed_page()` instead of doing the transform before merging.

Unless I misunderstood you, it is already done that way: since most of the examples on this page use `merge_page()`, I started with a naïve solution using this, that does not work. I then explain why it does not work, and I go on with the right solution using `merge_transformed_page()`.

> The advantage is that you can reuse the page. we could keep the merge_page for the watermarking only. Also maybe you could propose to update the example for underlay watermark using the over=False new option.

This page? https://pypdf.readthedocs.io/en/stable/user/add-watermark.html?highlight=watermark
I can update it.

**Comment by pubpub-zz** [comment]
> > @paternal Thanks for you contribution. I would more likely recommend to use `.merge_transformed_page()` instead of doing the transform before merging.
> 
> Unless I misunderstood you, it is already done that way: since most of the examples on this page use `merge_page()`, I started with a naïve solution using this, that does not work. I then explain why it does not work, and I go on with the right solution using `merge_transformed_page()`.

I did not read correctly your proposal with the diff mod of github 🤭
I've seen so many people using transformation, that is currently not applying any changes on the annots.

> 
> > The advantage is that you can reuse the page. we could keep the merge_page for the watermarking only. Also maybe you could propose to update the example for underlay watermark using the over=False new option.
> 
> This page? https://pypdf.readthedocs.io/en/stable/user/add-watermark.html?highlight=watermark I can update it.

yes🙂

File: docs/user/cropping-and-transforming.md [hunk 1]
```diff
@@ -205,3 +205,86 @@ In the mean time, you can add the following code to get the old behavior:
 ```python
 pypdf._page.MERGE_CROP_BOX = "trimbox"
 ```
+
+# Transforming several copies of the same page
+
+We have designed the following business card (A8 format) to advertize our new startup.
```
Comment (MartinThoma) [inline docs/user/cropping-and-transforming.md:211]: hahaha, I love it :smile: :heart:

**Comment by MartinThoma** [comment]
@paternal Thank you :pray: 

If you want, I can add you to the contributors page: https://pypdf.readthedocs.io/en/latest/meta/CONTRIBUTORS.html
Just let me know which name I should use :-) (just Louis? paternal?)

**Comment by paternal** [comment]
> Just let me know which name I should use :-) (just Louis? paternal?)

Louis Paternault @paternal 

Thanks

**Comment by MartinThoma** [comment]
done :-)

**Pull request merged**